### PR TITLE
Update ann_layer_test.cpp

### PR DESCRIPTION
**Pull request by GauravSarkar**
changes made in matrices and vector input.changes made with respect to issue #2699

### CODE REVIEW

File: src/mlpack/tests/ann_layer_test.cpp [hunk 1]
```diff
@@ -2678,31 +2688,31 @@ TEST_CASE("AtrousConvolutionLayerPaddingTest", "[ANNLayerTest]")
...
+           {0, -1.1355},
+           {-1.2247 , -0.1622} };
   CheckMatrices(output, result, 1e-1);
   result.clear();
 
   output = model.Mean();
-  result << 4.9000 << 3.2333 << arma::endr;
-
+ 
+ result ={ 4.9000,3.233};
```
Comment (shrit) [inline src/mlpack/tests/ann_layer_test.cpp:2709]:
```suggestion
 result ={ 4.9000, 3.233};
```

File: src/mlpack/tests/ann_layer_test.cpp [hunk 2]
```diff
@@ -2077,19 +2082,24 @@ TEST_CASE("BatchNormTest", "[ANNLayerTest]")
   model.Forward(input, output);
 
   // Values calculated using torch.nn.BatchNorm2d(momentum = None).
-  result << 0.9521 << 0.0898 << -1.0419 << arma::endr
-         << 1.0273 << -0.0571 << -0.9702 << arma::endr
-         << 0.9586 << 0.0783 << -1.0368 << arma::endr;
+
+result = { {0.9521 , 0.0898 , -1.0419 },
```
Comment (shrit) [inline src/mlpack/tests/ann_layer_test.cpp:2086]:
```suggestion
result = { {0.9521, 0.0898, -1.0419 },
```

**Review by shrit — COMMENTED** [review]
@GauravSarkar Can you fix styles? everywhere? as detailed in the suggestions. Thanks

**Comment by GauravSarkar** [comment]
Yes definetely, I will fix the styles. Thanks for reviewing.

File: src/mlpack/tests/ann_layer_test.cpp (hunk 1 above)
Comment (conradsnicta) [reply src/mlpack/tests/ann_layer_test.cpp:2709]: @GauravSarkar No need to rush. Make sure the layout is nice and clean. This helps with maintaining the code.  
Ensure the spacing is consistent, columns line up, etc.  Examples:

    X = { 4.9000, 3.233 };

Note the space after `=`, after `{`, after `,`, and before `}`.

    X = { { 5.1, 3.5, 1.4 },
          { 4.9, 3.0, 1.4 },
          { 4.7, 3.2, 1.3 } };

Note how the columns line up.

Comment (GauravSarkar) [reply src/mlpack/tests/ann_layer_test.cpp:2709]: input = { { 1,  446, 42 },
            { 2,  16,  63 },
            { 3,  13,  63 },
            { 4,  21,  21 },
            { 1,  13,  11 },
            { 32, 45,  42 },
            { 22, 16,  63 },
            { 32, 13,  42 } };
Is this layout correct?

**Comment by GauravSarkar** [comment]
@shrit  I have made the required changes

File: src/mlpack/tests/ann_layer_test.cpp (hunk 1 above)
Comment (zoq) [reply src/mlpack/tests/ann_layer_test.cpp]: If it follows the format Conrad posted above it's correct.

File: src/mlpack/tests/ann_layer_test.cpp [hunk 3]
```diff
@@ -4389,60 +4405,42 @@ TEST_CASE("SpatialDropoutLayerTest", "[ANNLayerTest]")
   SpatialDropout<> module(3, 0.2);
 
   // Input is a batch of 2 images, each of size (2,2) and having 4 channels.
-  input << 0.4963 << 0.0885 << 0.7682 << 0.1320 << 0.3074 << 0.4901 << 0.6341
-      << 0.8964 << 0.4556 << 0.3489 << 0.6323 << 0.4017 << arma::endr;
+  input = { 0.4963 , 0.0885 , 0.7682 , 0.1320 , 0.3074 , 0.4901 , 0.6341 , 0.8964 , 0.4556 , 0.3489 , 0.6323 , 0.4017 };
```
Comment (Abilityguy) [inline src/mlpack/tests/ann_layer_test.cpp:4408]:
```suggestion
  input = { 0.4963 , 0.0885 , 0.7682 , 0.1320 , 0.3074 , 0.4901 , 0.6341 ,
      0.8964 , 0.4556 , 0.3489 , 0.6323 , 0.4017 };
```
There seem to be few lines which are longer than 80 characters. 
You could continue them in a new line. 

For more details on this, refer the style guide 
https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping

File: src/mlpack/tests/ann_layer_test.cpp [hunk 4]
```diff
@@ -2027,9 +2027,10 @@ TEST_CASE("BatchNormTest", "[ANNLayerTest]")
 
   // Value calculates using torch.nn.BatchNorm2d(momentum = None).
   arma::mat result;
-  result << 1.1658 << 0.1100 << -1.2758 << arma::endr
-         << 1.2579 << -0.0699 << -1.1880 << arma::endr
-         << 1.1737 << 0.0958 << -1.2695 << arma::endr;
+
```
Comment (Abilityguy) [inline src/mlpack/tests/ann_layer_test.cpp:2030]:
```suggestion
```
I don't think this new line is required. There are other places with an extra newline.

File: src/mlpack/tests/ann_layer_test.cpp [hunk 5]
```diff
@@ -2039,36 +2040,38 @@ TEST_CASE("BatchNormTest", "[ANNLayerTest]")
...
   CheckMatrices(output, result, 1e-1);
 
   // Values calculated using torch.nn.BatchNorm2d().
   output = model2.TrainingMean();
-  result << 0.3333 << arma::endr
-         << 0.3100 << arma::endr
-         << 0.3067 << arma::endr;
-
+  
+result = { { 0.3333 },
```
Comment (Abilityguy) [inline src/mlpack/tests/ann_layer_test.cpp:2053]:
```suggestion
  result = { { 0.3333 },
```
This initialisation and a few others are not indented properly.

**Review by Abilityguy — CHANGES_REQUESTED** [review]
Hi @GauravSarkar  , there are few more style fixes that are needed to pass the style checks.
I have marked a few of them below.

File: src/mlpack/tests/ann_layer_test.cpp (hunk 5 above)
Comment (GauravSarkar) [reply src/mlpack/tests/ann_layer_test.cpp:2053]: thanks @Abilityguy  .I will resolve the problem soon.

**Comment by GauravSarkar** [comment]
@Abilityguy I have made the required style changes.

File: src/mlpack/tests/ann_layer_test.cpp [hunk 6]
```diff
@@ -2039,36 +2037,37 @@ TEST_CASE("BatchNormTest", "[ANNLayerTest]")
 
   // Values calculated using torch.nn.BatchNorm2d(momentum = None).
   output = model.TrainingMean();
-  result << 3.33333333 << arma::endr
-         << 3.1 << arma::endr
-         << 3.06666666 << arma::endr;
+
```
Comment (Abilityguy) [inline src/mlpack/tests/ann_layer_test.cpp:2040]:
```suggestion

```

This is another empty line with a whitespace.

File: src/mlpack/tests/ann_layer_test.cpp (hunk 3 above)
Comment (Abilityguy) [reply src/mlpack/tests/ann_layer_test.cpp:4390]: Good job on the line wrapping. You missed out the line pointed out here.

File: src/mlpack/tests/ann_layer_test.cpp [hunk 7]
```diff
@@ -2678,31 +2677,29 @@ TEST_CASE("AtrousConvolutionLayerPaddingTest", "[ANNLayerTest]")
...
+  result = { {  1.2247,   1.2978 },
+             {  0,       -1.1355 },
+             { -1.2247,  -0.1622 } };
   CheckMatrices(output, result, 1e-1);
   result.clear();
 
   output = model.Mean();
-  result << 4.9000 << 3.2333 << arma::endr;
-
+
```
Comment (Abilityguy) [inline src/mlpack/tests/ann_layer_test.cpp:2695]:
```suggestion
```
Extra line here.

File: src/mlpack/tests/ann_layer_test.cpp [hunk 8]
```diff
@@ -4074,15 +4069,15 @@ TEST_CASE("BatchNormWithMinBatchesTest", "[ANNLayerTest]")
...
-
+  result = { { -0.12195,  11.20426,  0.92158 },
+             { -0.0965,   0.259824,  1.4560  },
+             { -0.071054, 0.183567,  1.45607 },
+             { -0.045601, 0.3870852, 0.38708 },
+             { -0.305288, 1.7683,    1.4227  },
+             {  5.05166,  7.29812,   6.7797  },
+             {  3.323614, 2.2867,    10.4086 },
+             {  5.05166,  1.7683,    6.7797  } };
+
```
Comment (Abilityguy) [inline src/mlpack/tests/ann_layer_test.cpp:4080]:
```suggestion

```
Extra whitespaces in this line.

File: src/mlpack/tests/ann_layer_test.cpp [hunk 9]
```diff
@@ -4095,20 +4090,20 @@ TEST_CASE("BatchNormWithMinBatchesTest", "[ANNLayerTest]")
...
-      << 1.894857 << -0.901267 << arma::endr
-      << -0.980402 << 0.180253 << arma::endr;
-
+  input = { { 12,  443 },
+            { 134, 45  },
+            { 11,  13  },
+            { 14,  55  },
+            { 110, 4   },
+            { 1,   45  } };
+
```
Comment (Abilityguy) [inline src/mlpack/tests/ann_layer_test.cpp:4099]:
```suggestion

```
Extra whitespaces here.

File: src/mlpack/tests/ann_layer_test.cpp [hunk 10]
```diff
@@ -4095,20 +4090,20 @@ TEST_CASE("BatchNormWithMinBatchesTest", "[ANNLayerTest]")
...
+            { 110, 4   },
+            { 1,   45  } };
+  
+  result = { { -0.629337,  2.14791   },
+             {  0.156797, -0.416694  },
+             { -0.63578,  -0.622893  },
+             { -0.637481,  0.4440386 },
+             {  1.894857, -0.901267  },
+             { -0.980402,  0.180253  } };
+
```
Comment (Abilityguy) [inline src/mlpack/tests/ann_layer_test.cpp:4106]:
```suggestion

```
Extra whitespaces here.

**Review by Abilityguy — CHANGES_REQUESTED** [review]
Hi @GauravSarkar , there are still a few style changes to be made to pass the style checks. I have pointed a few of them out. Most of these issues are due to having extra new lines or extra whitespaces in an empty line. Fixing this should pass the style checks.

**Comment by Abilityguy** [comment]
Hey @GauravSarkar, have you tested your code? It seems like there are a few errors in your code.

Try building your code by going to the build folder and running the ```cmake ..``` and then the ```make install``` commands. 
This should help you in identifying errors in your code.

**Comment by Abilityguy** [comment]
Hi @GauravSarkar , Will you be taking this up?
I believe a few more changes and this will be ready for merge.

**Pull request closed without merging**